### PR TITLE
added de-DE translations

### PR DESCRIPTION
**Pull request by devofthings**
like the title says I added support for de-DE. In order to add more languages in the future we may should consider sorting them alphabetically in the i18n file. I can do that with the current ones if you like. Also a contribution guide would be awesome :D

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -208,6 +208,31 @@ const resources = {
       select: 'Вибір мови',
     },
   },
+  'de-DE': {
+    translation: {
+      title: 'WiFi Card',
+      'desc.use':
+        'Druck dir eine simple Karte mit deinen WiFi Anmeldedaten. Klebe sie an deinen Kühlschrank, behalte sie in deinem Portemonnaie, etc.',
```
Comment (Andanan) [inline src/i18n.js:215]:
```suggestion
        'Druck dir eine simple Karte mit deinen WLAN-Anmeldedaten aus. Klebe sie an deinen Kühlschrank, behalte sie in deinem Geldbeutel, etc.',
```

File: src/i18n.js [hunk 2]
```diff
@@ -208,6 +208,31 @@ const resources = {
...
+      'desc.privacy':
+        'Deine WiFi Anmeldedaten werden niemals zum Server gesendet. Es gibt kein Tracking, Analytics, oder Fingerprinting auf dieser Website. Hier geht es zum',
+      'desc.source': 'Source Code',
```
Comment (Andanan) [inline src/i18n.js:218]:
```suggestion
      'desc.source': 'Quellcode',
```

File: src/i18n.js [hunk 3]
```diff
@@ -208,6 +208,31 @@ const resources = {
...
+      'wifi.login': 'WiFi Login',
```
Comment (Andanan) [inline src/i18n.js:219]:
```suggestion
      'wifi.login': 'WLAN-Anmeldedaten',
```

File: src/i18n.js [hunk 4]
```diff
@@ -208,6 +208,31 @@ const resources = {
...
+      'wifi.name': 'Netzwerk Name',
+      'wifi.name.placeholder': 'WiFi Netzwerk Name',
```
Comment (Andanan) [inline src/i18n.js:221]:
```suggestion
      'wifi.name.placeholder': 'WLAN-Netzwerkname (SSID)',
```

File: src/i18n.js [hunk 5]
```diff
@@ -208,6 +208,31 @@ const resources = {
...
+      'wifi.password': 'Passwort',
+      'wifi.password.placeholder': 'Passwort',
+      'wifi.password.hide': 'Passwordfeld vor dem Drucken ausblenden',
```
Comment (Andanan) [inline src/i18n.js:224]: Fix typo

```suggestion
      'wifi.password.hide': 'Passwortfeld vor dem Drucken ausblenden',
```

**Review by Andanan — CHANGES_REQUESTED** [review]
In my opinion some of the terms you use here, are not the best translation. For example `WiFi` is far less often used than `WLAN` and `Portmonaie` seems very old-fashioned. I guess `Geldbeutel` would be a better fit.

Furthermore I'm not sure whether `Anmeldedaten` is the best term (in my experience `Zugangsdaten` is used more often).  
I kept it as it was, because that is probably just my personal preference.

I provided some enhancements you should consider.

File: src/i18n.js [hunk 6]
```diff
@@ -208,6 +208,31 @@ const resources = {
       select: 'Вибір мови',
     },
   },
+  'de-DE': {
+    translation: {
+      title: 'WiFi Card',
+      'desc.use':
+        'Druck dir eine simple Karte mit deinen WiFi Anmeldedaten. Klebe sie an deinen Kühlschrank, behalte sie in deinem Portemonnaie, etc.',
+      'desc.privacy':
+        'Deine WiFi Anmeldedaten werden niemals zum Server gesendet. Es gibt kein Tracking, Analytics, oder Fingerprinting auf dieser Website. Hier geht es zum',
```
Comment (Andanan) [inline src/i18n.js:217]: Missed that one:

```suggestion
        'Deine WLAN-Anmeldedaten werden niemals zum Server gesendet. Es gibt kein Tracking, Analytics, oder Fingerprinting auf dieser Website. Hier geht es zum',
```

File: src/i18n.js (hunk 2 above)
Comment (devofthings) [reply src/i18n.js:218]: Ahh yes the good old fashioned denglish problem i will add this and some of the other suggestions ✌🏼

File: src/i18n.js [hunk 7]
```diff
@@ -208,6 +208,31 @@ const resources = {
...
+  'de-DE': {
+    translation: {
+      title: 'WLAN Karte',
+      'desc.use':
+        'Druck dir eine simple Karte mit deinen WLAN-Zugangsdaten aus. Klebe sie an deinen Kühlschrank, behalte sie in deinem Portemonnaie, etc.',
+      'desc.privacy':
+        'Deine Zugangsdaten werden niemals zum Server gesendet. Es gibt kein Tracking, Analytics, oder Fingerprinting auf dieser Website. Hier geht es zum',
+      'desc.source': 'Quellcode',
+      'wifi.login': 'WLAN-Zugangsdaten',
+      'wifi.name': 'Netzwerk Name',
```
Comment (Andanan) [inline src/i18n.js:220]:
```suggestion
      'wifi.name': 'WLAN-Netzwerkname (SSID)',
```

File: src/i18n.js [hunk 8]
```diff
@@ -208,6 +208,31 @@ const resources = {
...
+      'wifi.name.placeholder': 'WLAN-Netzwerkname (SSID)',
```
Comment (Andanan) [inline src/i18n.js:221]:
```suggestion
      'wifi.name.placeholder': 'WLAN-Netzwerkname',
```

**Review by Andanan — CHANGES_REQUESTED** [review]
I'm Sorry, i must have mixed up the placeholder and the normal label for `wifi.name` when i made my suggestions 😱

But these should really be the last changes now.

**Comment by bndw** [comment]
@devofthings Thanks for the suggestion on the language guide, tracking in https://github.com/bndw/wifi-card/issues/117

**Comment by bndw** [comment]
@devofthings Mind rebasing master to resolve conflicts, then we should be good to go?

**Comment by Andanan** [comment]
I just commented in #118. That PR seems to be more current than this one and it contains the same translations as this so it would probably be better for the merge.

**Comment by devofthings** [comment]
@Andanan & @bndw  I just merged the master inside this branch, there are no more merge conflicts :)

**Review by Baoyuantop — APPROVED** [review]

**Comment by bndw** [comment]
Completed in https://github.com/bndw/wifi-card/pull/140

**Pull request closed without merging**